dfc/tests/regression_test.go

```diff
@@ -92,6 +92,119 @@ var (
 	failLRU = ""
 )
 
+// Note: This test is most effective when multiple targets are deployed.
```

Comment:
You can add check for targetCount > 1 in the test and generate t.Warning if targets == 1 :) So, people will be informed that the test is not reliable